spec/support/it_accepts_metadata_json_targets.rb

```diff
@@ -17,28 +17,29 @@
         let(:globbed_files) { [module_metadata_json] }
 
         it 'returns the path to metadata.json in the module' do
-          expect(parsed_targets).to eq(globbed_files)
+          expect(parsed_targets.first).to eq(globbed_files)
         end
       end
 
       context 'and the module does not contain a metadata.json file' do
         it 'returns no targets' do
-          expect(parsed_targets).to eq([])
+          expect(parsed_targets.first).to eq([])
         end
       end
     end
 
-    context 'when given specific target files' do
+    context 'when given unmatched target files' do
       let(:targets) { ['target1', 'target2.json'] }
 
       before(:each) do
         targets.each do |target|
           allow(File).to receive(:directory?).with(target).and_return(false)
+          allow(File).to receive(:file?).with(target).and_return(true)
         end
       end
 
-      it 'returns the targets' do
-        expect(parsed_targets).to eq(targets)
+      it 'skips the targets' do
+        expect(parsed_targets.first).to eq([])
```

Comment:
Might be valuable to also put an expectation on the _skipped_ and _invalid_ values